### DISCUSSION TITLE
Question about the template for the dissertation

### DISCUSSION THREAD

**Discussion by dansyndicate** [joaomlourenco/novathesis#483] [Q&A]
Good evening professor, i need help with the template for the dissertation document. 

I have the following commands:
  \ntsetup{doctype=msc}
  \ntsetup{docstatus=provisional} 
  \ntsetup{media=paper} 

1. First i would like to remove the setence "This document was created with the (pdf/Xe/Lua)LATEX processor and the NOVAthesis template (v7.1.5) [1]." from this page and i don't know how.

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/9fcbb366-d165-4d7f-bd69-33f31e263bfd)

2. Then i want to remove this line at the end of the document. 

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/8ee7c8f9-524d-43d8-8541-866a61611af5)

![image](https://github.com/joaomlourenco/novathesis/assets/129518837/4dbde66a-3704-450c-9a2a-202edc1b538f)

Can you help me?

Best regards,
Daniel Martins

**Comment by joaomlourenco** [comment]
Dear Daniel,

1) Why do you want to remove that line?  Don't you think that the person (me) who spent hundreds of hours developing a template for you to use for free, deserves the recognition?  Please be kind and make a donation, and I'll remove that line.

2) That is not a line!  That is the book spine (_lombada_).  Is should be there, but you can force it off with the option `\ntsetup{spine/layout=no}`.

**Reply by dansyndicate** [reply]
1. I am sorry professor, I just saw a document that didn't have that sentence so I thought it was possible to remove to remove it. I will keep it. 
2. Ok thank you professor
